docs/release-info/release-notes.md

```diff
@@ -57,50 +57,745 @@ For tips about how to write a good release note, see [Release notes](https://git
+
+In addition, there are new metrics available to monitor the performance of centralized schema management:
+
+- `metadatacache/schemaPoll/count`
+- `metadatacache/schemaPoll/failed`
+- `metadatacache/schemaPoll/time`
+- `metadacache/init/time`
+- `metadatacache/refresh/count`
+- `metadatacache/refresh/time`
+- `metadatacache/backfill/count`
+- `metadatacache/finalizedSegmentMetadata/size`
+- `metadatacache/finalizedSegmentMetadata/count`
+- `metadatacache/finalizedSchemaPayload/count`
+- `metadatacache/temporaryMetadataQueryResults/count`
+- `metadatacache/temporaryPublishedMetadataQueryResults/count`
+
+For more information, see [Metrics](../operations/metrics.md).
+
+[#15817](https://github.com/apache/druid/pull/15817)
+
+Also, note the following changes to the default values of segment schema cleanup:
+
+* The default value for `druid.coordinator.kill.segmentSchema.period` has changes from `PT1H` to `P1D`.
+* The default value for `druid.coordinator.kill.segmentSchema.durationToRetain` has changed from `PR6H` to `P90D`.
+
+[#16354](https://github.com/apache/druid/pull/16354)
+
+### MSQ support for window functions
+
+Added support for using window functions with the MSQ task engine as the query engine.
+
+[#15470](https://github.com/apache/druid/pull/15470)
+
+### MSQ support for Google Cloud Storage
+
+You can now export MSQ results to a Google Cloud Storage (GCS) path by passing the function `google()` as an argument to the `EXTERN` function.
+
+[#16051](https://github.com/apache/druid/pull/16051)
+
+### RabbitMQ extension
+
+A new RabbitMQ extension is available as a community contribution.
+The RabbitMQ extension (`druid-rabbit-indexing-service`) lets you manage the creation and lifetime of rabbit indexing tasks. These indexing tasks read events from [RabbitMQ](https://www.rabbitmq.com) through [super streams](https://www.rabbitmq.com/docs/streams#super-streams).
+
+As super streams allow exactly once delivery with full support for partitioning, they are compatible with Druid's modern ingestion algorithm, without the downsides of the prior RabbitMQ firehose.
+
+Note that this uses the RabbitMQ streams feature and not a conventional exchange. You need to make sure that your messages are in a super stream before consumption. For more information, see [RabbitMQ documentation](https://www.rabbitmq.com/docs).
+
+[#14137](https://github.com/apache/druid/pull/14137)
+
 ## Functional area and related changes
 
 This section contains detailed release notes separated by areas.
 
 ### Web console
 
+#### Search in tables and columns
+
+You can now use the **Query** view to search in tables and columns.
+
+![Use the sidebar to search in tables and columns in Query view](./assets/30.0.0-console-search.png)
+
+[#15990](https://github.com/apache/druid/pull/15990)
+
+#### Improved array ingestion UX
```

Comment:
This was already shipped as part of Druid 29.0.1 does it make sense to talk about this here again?

Reply:
PR 15927 should not have been included in the Druid 30.0.0 release notes. Removed.